src/test/java/io/managed/services/test/rhoas/KafkaRhoasBasicTests.java

```diff
@@ -517,6 +521,40 @@ public void testDeleteServiceAccount() {
             () -> cli.describeServiceAccount(serviceAccount.getClientId()));
     }
 
+    @Test(dependsOnMethods = "testApplyKafkaInstance", enabled = true)
+    @SneakyThrows
+    public void testUpdateKafkaOwner() {
+        try
+        {
+            cli.UpdateKafkaOwner(Environment.SECONDARY_USERNAME, KAFKA_INSTANCE_NAME);
+            var k = cli.describeKafkaByName(KAFKA_INSTANCE_NAME);
+            LOGGER.debug(k);
+            assertEquals(Environment.SECONDARY_USERNAME, k.getOwner());
+        }
+        catch (Throwable t) {
+            LOGGER.error("testUpdateKafkaOwner ERROR: ", t);
+        }
+        finally {
+            LOGGER.info("login the CLI with the Secondary User (owner of the instance)");
+            CLIUtils.login(vertx, cli, Environment.SECONDARY_USERNAME, Environment.SECONDARY_PASSWORD).get();
+            cli.UpdateKafkaOwner(Environment.PRIMARY_USERNAME, KAFKA_INSTANCE_NAME);
+            var k = cli.describeKafkaByName(KAFKA_INSTANCE_NAME);
+            LOGGER.debug(k);
+            assertEquals(Environment.PRIMARY_USERNAME, k.getOwner());
+            LOGGER.info("login the CLI with the Primary User (re-owner of the instance)");
+            CLIUtils.login(vertx, cli, Environment.PRIMARY_USERNAME, Environment.SECONDARY_PASSWORD).get();
```

Comment:
change this implementation so that you would not log in with cli (this is a must have in case you will solve recommendation regarding removing of SECONDARY_PASSWORD). 
I suggest that you will log in here with ADMIN_OFFLINE_TOKEN (TO REITERATRE i am not tlaking about cli but API) and simply delete kafka with admin, as admin is cabale of deleting it either way (if it is still held be primary user, or secondary one) plus you will not need to log in log back with primary user.

Reply:
for inspiration you can find how to log in to kafka admin api in other suites like one under kafka package. (all of them use it)

Reply:
I think that It wouldn't work because at the end, I need to use the PRIMARY user to delete that cluster.
Instead of that, what I've done is create an afterMethod (in: 8cd74dc) which executes just after "testUpdateKafkaOwner" and using the admin user and the API, restores the old owner of the instance. Is it OK?

Reply:
what exactly is problem why you think it should not work ? The only problem I can think of is in case you would use the same managed kafka instance here as in all other tests. In this case yes you would delete instance for other tests. 

Apart from that I am okay with whatever implementation you will prefer here.